tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -82,6 +82,11 @@ class CommandLineDrawingTestConfigImpl : public DrawingTestConfigImpl {
     virtual std::string GetComparisonPath() override {
         return _comparisonPath;
     }
+
+    virtual std::string GetResourcePath(const std::string& resource) override {
+        // Using / as a path separator is valid on both Windows and OS X.
+        return __ModuleDirectory() + "/data/" + resource;
```

Comment:
The Drawing test vcxproj absolutely copies a data/ directory.

If we standardize on `data/`, we don't have to propagate that decision into every test file consumer (and centralizing it here lets us change it in a single place if we do decide to.)